taigaProject/src/taigaApi/controller/project_controller.py

```diff
@@ -7,12 +8,12 @@
 
 router = APIRouter()
 
-@router.get("/Project")
-def auth(projectRequest: ProjectRequest):
-    project_info = get_project_by_slug(projectRequest.projectslug, projectRequest.authtoken)
+@router.post("/project")
+def auth(projectRequest: ProjectRequest, token: Annotated[str | None, Header()] = None):
```

Comment:
What is the need of this change? Is it for a front end change?

Reply:
Token are generally sent as part of header. That's why I made this change and removed the field from projectRequest.